test/loadtest/readme.md

```diff
@@ -0,0 +1,75 @@
+# Distributed load testing using Bedrock kubernetes Cluster and Locust
```

Comment:
Use title casing here: "Distributed Load Testing on Bedrock Kubernetes Clusters with Locust"

Reply:
Updated it.